docs/admin/searx.favicons.rst

```diff
@@ -0,0 +1,249 @@
+.. _favicons:
+
+========
+Favicons
+========
+
+.. sidebar:: warning
+
+   Don't activate the favicons before reading the documentation.
+
+.. contents::
+   :depth: 2
+   :local:
+   :backlinks: entry
+
+Activating the favicons in SearXNG is very easy, but this **generates a
+significantly higher load** in the communication between client and server and
+additional resources are used on the server.
```

Comment:
```suggestion
significantly higher load** in the communication between client and server and
increases resource usage on the server.
```
nit: two `and`s, grammar

Reply:
my intention was to say: *"higher load in the client/server communication and increases resource usage on the server."* --> [fixed](https://github.com/searxng/searxng/compare/a76b37fe75f50c8281fbec24499e74211ba2b67e..7c7482fc107484e883803df65176647d58aff989)